Paintroid/src/main/java/org/catrobat/paintroid/tools/implementation/TextTool.kt

```diff
@@ -140,25 +140,49 @@ class TextTool(
             override fun setFont(font: String) {
                 this@TextTool.font = font
                 updateTypeface()
+                val width = boxWidth
+                val height = boxHeight
+                val position = PointF(toolPosition.x, toolPosition.y)
                 createAndSetBitmap()
+                toolPosition.set(position)
+                boxWidth = width
+                boxHeight = height
             }
 
             override fun setUnderlined(underlined: Boolean) {
                 this@TextTool.underlined = underlined
                 textPaint.isUnderlineText = this@TextTool.underlined
+                val width = boxWidth
+                val height = boxHeight
+                val position = PointF(toolPosition.x, toolPosition.y)
                 createAndSetBitmap()
+                toolPosition.set(position)
+                boxWidth = width
+                boxHeight = height
```

Comment:
Could you please create a new function for these lines as these same lines are being duplicated in 3-4 methods?